dist/ramda.js

```diff
@@ -2184,7 +2184,7 @@
    * implementation.
    *
    * @private
-   * @param {Function} fn ramda implemtation
+   * @param {Function} fn ramda implementation
```

Comment:
This file is generated.  Could you do the same in `source/internal/_checkForMethod,js`?

Reply:
@CrossEye not seeing that spelling there:
https://github.com/ramda/ramda/blob/master/source/internal/_checkForMethod.js

Reply:
Hmm, well it should be fixed on the next release then.  But we prefer to never check in this generated file.

Reply:
@CrossEye yeah normally I would've left it alone but since it had diverged and there are instances where it was committed in between releases (see e.g., f508b04ee0ecac924eb41ca4d9ceadc13b8a3b14) I included it. In any case I'll drop it.